### PR TITLE
Add announcement of Trino Gateway

### PR DESCRIPTION
**Pull request by mosabua**
Still a couple of things to do:

* flesh out content ✅ 
* add image ✅ 
* link to project ✅ 
* get approval about mentioning contributors ✅ 
* organize review ✅ 
* update date just before publishing ✅ 

Also note the logo is preliminary only .. we are going to get a better one (or even some options) from the original designer of the Trino logo - @aliloney

### CODE REVIEW

File: _posts/2023-09-27-trino-gateway.md [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+
+Today, we are happy to announce our [first release of Trino
+Gateway](https://github.com/trinodb/trino-gateway/blob/main/docs/release-notes.md#trino-gateway-3-26-sep-2023).
+The release is the result of many, many months of effort to move the legacy
+Presto Gateway to Trino, start a refactor of the project, and add numerous new
+features.
+
+Many larger deployments across the Trino community rely on the gateway as a load
+balancer, proxy server, and configurable routing gateway for multiple Trino
+clusters. User don't need to worry about what catalog and data source is
```
Comment (martint) [inline _posts/2023-09-27-trino-gateway.md:29]: Users

File: _posts/2023-09-27-trino-gateway.md [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+Gateway. Some of these users created numerous improvements in isolation of each
+other, sometimes even implementing the same feature multiple times. This first
+release of Trino Gateway starts a strong collaboration of some of these users.
+Bloomberg contributed the main bulk of the new features, maintainers and
+contributors from Starburst pulled together the stakeholders and managed the
+project, and other collaborators from Apple, Naver, LinkedIn, Lyft, and Dune are
+already helping out and ready to move the project forward.
+
+There are exciting times ahead for the project, and we have big plans for
+documentation, installation , and general modernizations of the app, so go and
```
Comment (martint) [inline _posts/2023-09-27-trino-gateway.md:44]: Remove space before comma

**Review by martint — APPROVED** [review]

File: _posts/2023-09-27-trino-gateway.md [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+show_pagenav: false
+---
+
+You started with one Trino cluster, and your users like the power for SQL and
+[querying all sorts of data sources]({{site_url}}/ecosystem.html#data-sources).
+Then you needed to upgrade and got a cluster for testing going. That was a while
+ago, and now you run a separate cluster configured for ETL workloads with
+fault-tolerant execution, and some others with different configurations.
+
+With Trino Gateway we now have an answer to your users request to provide on URL
```
Comment (vishalya) [inline _posts/2023-09-27-trino-gateway.md:16]: I would also add a usecase of cluster availability during upgrades and switching to an upgrades cluster with ease.

File: _posts/2023-09-27-trino-gateway.md [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+
+Up to now, many users had to maintain separate forks of the legacy Presto
+Gateway. Some of these users created numerous improvements in isolation of each
+other, sometimes even implementing the same feature multiple times. This first
+release of Trino Gateway starts a strong collaboration of some of these users.
+Bloomberg contributed the main bulk of the new features, maintainers and
+contributors from Starburst pulled together the stakeholders and managed the
+project, and other collaborators from Apple, Naver, LinkedIn, Lyft, and Dune are
+already helping out and ready to move the project forward.
+
```
Comment (vishalya) [inline _posts/2023-09-27-trino-gateway.md:42]: are we mentioning the authentication/authorization features here? They are a major ask from users over the presto-gateway?

File: _posts/2023-09-27-trino-gateway.md (hunk 1 above)
Comment (mosabua) [reply _posts/2023-09-27-trino-gateway.md:29]: Updated

File: _posts/2023-09-27-trino-gateway.md (hunk 4 above)
Comment (mosabua) [reply _posts/2023-09-27-trino-gateway.md:42]: Done

File: _posts/2023-09-28-trino-gateway.md [hunk 5]
```diff
@@ -0,0 +1,69 @@
...
+show_pagenav: false
+---
+
+You started with one Trino cluster, and your users like the power for SQL and
+[querying all sorts of data sources]({{site_url}}/ecosystem/index.html#data-sources).
+Then you needed to upgrade and got a cluster for testing going. That was a while
+ago, and now you run a separate cluster configured for ETL workloads with
+fault-tolerant execution, and some others with different configurations.
+
+With Trino Gateway we now have an answer to your users request to provide on URL
```
Comment (xkrogen) [inline _posts/2023-09-28-trino-gateway.md:16]: typo :)
```suggestion
With Trino Gateway we now have an answer to your users request to provide one URL
```

Comment (mosabua) [reply _posts/2023-09-28-trino-gateway.md:16]: Already fixed

**Pull request merged**